ensime_shared/editor.py

```diff
@@ -348,11 +346,8 @@ def cursorhold(self):
             class since it represents "the Vim plugin" as an entity, iterating
             through its list of clients and calling the logic for each.
         """
-        # Make sure no plugin overrides this
-        self._vim.options['updatetime'] = 1000
-        # Keys with no effect, just retrigger CursorHold
-        # http://vim.wikia.com/wiki/Timer_to_execute_commands_periodically
-        self._vim.command(r'call feedkeys("f\e")')
+        self._vim.eval("timer_start({}, 'EnRefreshMessages', {{'repeat': -1}})"
+                       .format(500))
```

Comment:
Why the half of the time of before? Have you tried different values and this is the most suitable?
Also why the time as parameter and everything else inline?